client/lib/post-normalizer/rule-create-better-excerpt.js

```diff
@@ -90,8 +93,11 @@ export default function createBetterExcerpt( post ) {
 		return post;
 	}
 
-	post.better_excerpt = formatExcerpt( post.content );
-	post.better_excerpt_no_html = stripHTML( post.better_excerpt );
+	const strippedDom = buildStrippedDom( post.content );
+	post.content_no_html = striptags( strippedDom );
```

Comment:
I don't think I like the interdependencies here.  
is `post.content_no_html = striptags( post.content );` equivalent (that way it isn't dependent on `strippedDom`?

Are `striptags` and `stripHTML` essentially equivalent? If so I'd rather we lean towards using `striptags` everywhere instead of switching off between the two.